framework/resources/Datasources/tpcds/maprdb/json/importTables.sh

```diff
@@ -10,138 +10,138 @@ if [ -n $USERNAME ]
 then
     user=$USERNAME
 else
-    user=`ps -aef | grep Drillbit | grep org.apache.drill | cut -d' ' -f1 | head -1`
+    user=ps -aef | grep Drillbit | grep org.apache.drill | cut -d' ' -f1 | head -1
```

Comment:
Don't you need the escape tics? Without that, the shell interpreter will assume the words after `ps` is a command.

Reply:
You are right, will fix it.